### PR TITLE
Clarify hut block placement

### PR DESCRIPTION
**Pull request by DemSec**
Added instructions on how to place hut blocks for schematics.

Closes #

## Changes proposed:
- 
-

### CODE REVIEW

**Comment by Thodor12** [comment]
Can you utilize the system as created in #930, that's what I made it for :D

**Comment by DemSec** [comment]
> Can you utilize the system as created in #930, that's what I made it for :D

Oh, I see, so there's a version selection on the navbar, I didn't even notice. I didn't know to look for it. The design of it looks good though, I might be one of the few who speed-read past it.

File: pages/src/source/tutorials/schematics.md [hunk 1]
```diff
@@ -119,7 +119,12 @@ The scans folder is where the schematics are saved after performing a scan using
 
 To create new schematics, there are some guidelines that you must follow: the scans MUST have the same footprint for each level of the hut; the scans must contain the hut's block, for example the Builder's Hut block for the [Builder's Hut](../../source/buildings/builder); the hut block need to be exactly at the same place and have the same rotation for each level
 
-[1.18]
+{% version "1.19" after=true %}
+Must enable blueprint mode in server config. If on singleplayer, it's recommended to use a Configured mod or similar to edit this property in game menu. Setting this property is not required before 1.19.
+{% endversion %}
+
+When placing a hut block, ensure you are not within a colony border. Shift-right-click to place.
```
Comment (Thodor12) [inline pages/src/source/tutorials/schematics.md:126]: Does this text apply to any version or only 1.18 like you had before?

**Review by Thodor12 — CHANGES_REQUESTED** [review]

File: pages/src/source/tutorials/schematics.md (hunk 1 above)
Comment (DemSec) [reply pages/src/source/tutorials/schematics.md:126]: I think this applies to all versions.

Comment (Thodor12) [reply pages/src/source/tutorials/schematics.md:126]: In that case, can you please put a link to the "Configured" mod like so: `[Configured](https://www.curseforge.com/minecraft/mc-mods/configured)` so it's clear this is a mod name and not a text typo

**Comment by MotionlessTrain** [comment]
It should be possible to shift-place hut blocks without that config setting in 1.19 too. That config setting is just necessary to use "Constructed" paste outside of colonies

File: pages/src/source/tutorials/schematics.md [hunk 2]
```diff
@@ -119,10 +119,10 @@ The scans folder is where the schematics are saved after performing a scan using
...
-[1.19]
-Must enable blueprint mode in server config. If on singleplayer, it's recommended to use a Configured mod or similar to edit this property in game menu. Setting this property is not required in 1.18.
+{% version "1.19" after=true %}
+Must enable blueprint mode in server config. If on singleplayer, it's recommended to use a Configured mod or similar to edit this property in game menu. Setting this property is not required before 1.19.
+{% endversion %}
 
-[1.18]
 When placing a hut block, ensure you are not within a colony border. Shift-right-click to place.
 
 The scans' filenames need to follow the naming convention: {StyleName}/{HutName}{HutLevel}.blueprint. For example, for the Builder's Huts with the MyOwn style, we would have:
```
Comment (MotionlessTrain) [inline pages/src/source/tutorials/schematics.md:128]: This text is still for 1.18 only. That was where the [1.18] tag was initially placed for

Comment (DemSec) [reply pages/src/source/tutorials/schematics.md:125]: New changes are ready.

**Comment by DemSec** [comment]
The wiki could use this information.

**Comment by Talyda** [comment]
Not sure if I did something wrong, but it says my review is "pending" - I didn't intend to review, I wanted to ask (and will here) 
Why are you recommending using a mod to change the config in SP? We've never recommended using a mod to do that, you just edit the config, same way you do on a server.

**Comment by DemSec** [comment]
> Not sure if I did something wrong, but it says my review is "pending" - I didn't intend to review, I wanted to ask (and will here) Why are you recommending using a mod to change the config in SP? We've never recommended using a mod to do that, you just edit the config, same way you do on a server.

That part has been removed, no longer relevant to this merge. 

The reason was that it's always safer to let players change properties through GUI than via raw text file where a misclick on the keyboard could completely corrupt the whole config file, without player noticing.

**Review by anonymous563 — APPROVED** [review]

**Pull request merged**